packages/server/src/db/linkedRows/linkUtils.ts

```diff
@@ -99,9 +99,15 @@ export async function getLinkDocuments(args: {
 }
 
 export function getUniqueByProp(array: any[], prop: string) {
-  return array.filter((obj, pos, arr) => {
-    return arr.map(mapObj => mapObj[prop]).indexOf(obj[prop]) === pos
-  })
+  const seen = new Set()
+  const ret = []
```

Comment:
NAB: Can we rename this variable, or maybe there is a way to return a set as an array in JS?

Reply:
It's not the set that gets returned, it's a filtered version of the original array passed in. How do you feel about `filteredArray`?

Reply:
Yeah got it! `filteredArray` is spot on then 👍